src/ToolBox/SOS/Strike/sosdocsunix.txt

```diff
@@ -25,12 +25,12 @@ Object Inspection                  Examining code and stacks
 -----------------------------      -----------------------------
 DumpObj (dumpobj)                  Threads (clrthreads)
 DumpArray                          ThreadState
-DumpStackObjects (dso)             IP2MD (ip2md)
-DumpHeap (dumpheap)                u (clru)
-DumpVC                             DumpStack (dumpstack)
-GCRoot (gcroot)                    EEStack (eestack)
-PrintException (pe)                ClrStack (clrstack) 
-                                   GCInfo
+DumpAsync                          IP2MD (ip2md)
```

Comment:
Minor nit: I usually put the lowercase alias in paras in the help like "DumpAsync (dumpasync)".

Reply:
Fixed